### ISSUE TITLE
Setting `FileSystemRoot` to `'/'` results in an error from AccessionDirectoryProcessor

### ISSUE THREAD

**Issue by tznind** [SMI/SmiServices#921]
```
Cannot find the FileSystemRoot specified in the given MicroservicesOptions ()
[tznind@xxx smi-services-v4.0.0-linux-x64]$ ./smi dicom-directory-processor -d /fs-de-01/NCCID
Bootstrapper -> Main called, constructing host
2021-10-07 05:45:59.1815| INFO|SmiLogging|Logging config loaded from /home/tznind/smi-services-v4.0.0-linux-x64/Smi.NLog.config||
2021-10-07 05:45:59.1851| INFO|DicomDirectoryProcessorHost|Host logger created||
2021-10-07 05:45:59.1863| INFO|DicomDirectoryProcessorHost|Starting DicomDirectoryProcessor (Host=xxx PID=148718 User=tznind)||
System.ArgumentException: Cannot find the FileSystemRoot specified in the given MicroservicesOptions ()
   at Applications.DicomDirectoryProcessor.Execution.DicomDirectoryProcessorHost..ctor(GlobalOptions globals, DicomDirectoryProcessorCliOptions cliOptions) in /home/vsts/work/1/s/src/applications/Applications.DicomDirectoryProcessor/Execution/DicomDirectoryProcessorHost.cs:line 35
   at Applications.DicomDirectoryProcessor.Program.<>c__DisplayClass1_0.<OnParse>b__0() in /home/vsts/work/1/s/src/applications/Applications.DicomDirectoryProcessor/Program.cs:line 36
   at Smi.Common.Execution.MicroserviceHostBootstrapper.Main() in /home/vsts/work/1/s/src/common/Smi.Common/Execution/MicroserviceHostBootstrapper.cs:line 39
```

**Comment by tznind** [comment]
> Just curious - in your original example should the filesystem root not be set to `/fs-de-01/NCCID`? You can compare with the config from the NSH which has `/<stuff>/PACS` as the root, with the DICOM in various subdirectories under there.

Yes that would certainly be an option.  But we might end up with other mounts e.g. `/fs-de-02/SomeOtherImagingProj` so root may make sense.  But either way we need to fix this bug.  Can decide later on how to handle deployment/config.  

Presumably could get the IT team to mount to `/remotemounts/fs-de-01` but again I think allowing maximum flexibility for users of the software is what we want.

**Comment by jas88** [comment]
> Just curious - in your original example should the filesystem root not be set to `/fs-de-01/NCCID`? You can compare with the config from the NSH which has `/<stuff>/PACS` as the root, with the DICOM in various subdirectories under there.

For our use case there's no guarantee it'll be /fs-de-01 anything - indeed right now we need it to ingest both /fs-de-01/NCCID and /appimg-de-01/PACS (and a third path TBD): the only sane "root" in this deployment is indeed the actual root unless we start re-mapping things around as an ugly hack.

In any case, we should only strip "trailing" / or \ characters (e.g. "/data/" => "/data"), stripping a lone "/" down to an empty string isn't the right outcome in any scenario AFAICS, so I've patched it for now in PR #976
